src/test/java/org/candlepin/controller/EntitlerTest.java

```diff
@@ -487,10 +486,10 @@ public void testRevokesLapsedUnmappedGuestEntitlementsOnAutoHeal() throws Except
 
         p1.setEntitlements(Set.of(e1));
 
-        CandlepinQuery cqmock = mock(CandlepinQuery.class);
-        when(cqmock.iterator()).thenReturn(Collections.singletonList(e1).iterator());
+        List queryList = mock(List.class);
```

Comment:
We don't need to (nor want to) mock a list. Just return a proper list straight away and avoid all the pain that comes with mocking it out.

Note that this applies to all the areas in which this was done.